src/DependencyInjection/Configuration.php

```diff
@@ -26,13 +26,13 @@
      */
     public function __construct($debug)
     {
-        $this->debug = (Boolean)$debug;
+        $this->debug = (bool) $debug;
```

Comment:
Not part of this PR, but IMHO it would be cleaner to type hint against boolean and move the type casting to [`ProophEventStoreExtension:34`](https://github.com/mablae/event-store-symfony-bundle/blob/a093cef432dd924cae2c52f136536fb4f6068458/src/DependencyInjection/ProophEventStoreExtension.php#L34).